services/appflowy-collaborate/src/group/group_init.rs

```diff
@@ -200,7 +200,26 @@ impl CollabGroup {
       modified_at.elapsed().as_secs() > 60 * 60
     } else {
       let elapsed_secs = modified_at.elapsed().as_secs();
-      elapsed_secs > self.timeout_secs() && self.subscribers.is_empty()
+      if elapsed_secs > self.timeout_secs() {
+        // Mark the group as inactive if it has been inactive for more than 3 hours, regardless of the number of subscribers.
+        // Otherwise, return `true` only if there are no subscribers remaining in the group.
+        // If a client modifies a group that has already been marked as inactive (removed),
+        // the client will automatically send an initialization sync to reinitialize the group.
+        const MAXIMUM_SECS: u64 = 3 * 60 * 60;
```

Comment:
You don't need to use `u64`: Rust has `Duration` type specifically for type calculation, and `Duration::from_secs` is const fn so can be used with const values.

Reply:
using `MAXIMUM_SECS` to compare with  `elapsed_secs`. So create an const instance here instead of using 3 * 60 * 60.

Reply:
`modified_at.elapsed()` already returns `Duration` and durations can be compared against each other as well.